DockerRun.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+    This script can be used to build and run the IoMT Azure Function code in a docker container.
```

Comment:
This is currently in the root?  We probably want this in a directory.  How about a docker dir under deploy?

Reply:
Thank you for moving the script but can we move the docker file itself to the folder as well?  Thanks!  Otherwise changes look great.